<ui version="4.0" >
 <class>GridSpaceDialog</class>
 <widget class="QDialog" name="GridSpaceDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Define the Grid Space</string>
  </property>
  <widget class="QDialogButtonBox" name="okCancelButtons" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>430</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>260</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title" >
    <string>Define the experiment space in meters</string>
   </property>
   <widget class="QLineEdit" name="xValue" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Width (X)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>20</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Height (Y)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="yValue" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>40</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>170</x>
      <y>20</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Depth (Z)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zValue" >
    <property name="geometry" >
     <rect>
      <x>170</x>
      <y>40</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="_ySpacing" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>90</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7" >
    <property name="geometry" >
     <rect>
      <x>170</x>
      <y>70</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Z Spacing</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="_xSpacing" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>90</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>70</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Y Spacing</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="_zSpacing" >
    <property name="geometry" >
     <rect>
      <x>170</x>
      <y>90</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>X Spacing</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>20</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Name for this experiment</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="spaceName" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>50</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="_symmetricalCheckBox" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>90</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>Symmetrical Grid Space</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>130</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title" >
    <string>Grid Geology</string>
   </property>
   <widget class="QRadioButton" name="radioButton" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>30</y>
      <width>90</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>Layered</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>60</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>Heterogeneous / Non-Layered</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okCancelButtons</sender>
   <signal>accepted()</signal>
   <receiver>GridSpaceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okCancelButtons</sender>
   <signal>rejected()</signal>
   <receiver>GridSpaceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
